docs/guides/using_gpu.md

```diff
@@ -7,10 +7,10 @@ a model is compiled for CUDA, executing it on a non-CUDA-enabled machine will ra
 
 ## Support
 
-| Feature     | Built-in models | Custom models | Deployment | DataFrame |
-| ----------- | --------------- | ------------- | ---------- | --------- |
-| GPU support | ✅              | ✅            | ✅         | ❌        |
-|             |                 |               |            |           |
+| Feature     | Built-in models | Deep NNs and LLMs | Deployment | DataFrame |
```

Comment:
Do we say "Deep NNs"? I thought it was either NN or DL

Reply:
yes, Deep NNs is fine